baseplate/context/memcache/lib.py

```diff
@@ -0,0 +1,108 @@
+from six import string_types
+import logging
+import zlib
+
+
+try:
+    import cPickle as pickle
+except ImportError:
+    # python3
+    import pickle
+
+try:
+    from cStringIO import StringIO
+except ImportError:
+    try:
+        from StringIO import StringIO
+    except ImportError:
+        # python3
+        from io import StringIO
+
+
+FLAG_PICKLE = 1 << 0
```

Comment:
perhaps these would be good as a proper Enum?

(see: https://docs.python.org/3/library/enum.html . a backwards compat shim is imported on py2 in _compat)